lib/tokenizers.js

```diff
@@ -80,6 +97,10 @@ export async function getCurrent() {
 
 export async function deprecateCurrent() {
   // mark the `current` tokenizer as deprecated
+
+  // the query of 'tokenizer.state' is properly indexed. No additional queries
+  // should be added here without ensuring that additional queries are properly
+  // indexed first.
```

Comment:
Again, I don't think this comment is helpful -- we'd need to put it everywhere in the code to be consistent and it would be overkill. Instead, people need to be educated about basic query / index hygiene just like they are generally already educated about good variable names and other coding hygiene practices. It also doesn't seem to even make sense (to me) to talk about adding "additional queries here".